layersvt/screenshot.cpp

```diff
@@ -739,6 +739,10 @@ VK_LAYER_EXPORT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL vkGetDeviceProcAddr(VkD
 
 VK_LAYER_EXPORT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL vkGetInstanceProcAddr(VkInstance instance, const char *funcName) {
     PFN_vkVoidFunction proc = intercept_core_instance_command(funcName);
```

Comment:
I will move them and change them to `assert(instance)`.  After `intercept_core_instance_command`, `instance` is always valid.